properties/P000055.md

```diff
@@ -13,7 +13,7 @@ refs:
 The topology on the space $X$ is induced by some complete metric $d$,
 that is, a metric for which every Cauchy sequence converges.
 
-A sequence $a_n$ is Cauchy provided for each distance $\epsilon$, there is
+A sequence $x_n$ is Cauchy provided for each distance $\epsilon$, there is
```

Comment:
```suggestion
A sequence $(x_n)_n$ is *Cauchy* provided for each distance $\epsilon>0$, there is
```
While we are improving this, we may as well make things more precise for newcomers.